tests/lib/network_helper.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from netaddr import *
+import re
+import ipaddress
+def mac_to_snmp_format(mac_addr):
+    if mac_addr != None:
+        try:
+            mac_tmp = re.sub("[\s+\.\!\-\:\/_,$%^*(+\"\')]+|[+()?【】“”！，。？、~@#￥%……&*（）]+',","",mac_addr)
```

Comment:
I've asked before, why are you doing this? I don't like you stripping out all this because it might causing us to pass and convert something we did not intend to convert. Please do the conversion you need and just that...

Reply:
Okay, so I only use what I want.